posts/alpine-stack-size.md

```diff
@@ -0,0 +1,155 @@
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/snippet_extractor.rb:31:in `extract_expression_lines_at'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:218:in `read_failed_lines'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:163:in `failure_slash_error_lines'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:150:in `block in failure_lines'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:149:in `tap'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:149:in `failure_lines'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:34:in `colorized_message_lines'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:240:in `formatted_message_and_backtrace'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:86:in `fully_formatted_lines'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/exception_presenter.rb:78:in `fully_formatted'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/notifications.rb:200:in `fully_formatted'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/notifications.rb:114:in `block in fully_formatted_failed_examples'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/notifications.rb:113:in `each'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/notifications.rb:113:in `each_with_index'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/notifications.rb:113:in `fully_formatted_failed_examples'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/formatters/base_text_formatter.rb:32:in `dump_failures'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:206:in `block in notify'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:205:in `each'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:205:in `notify'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:175:in `block in finish'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:191:in `close_after'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:171:in `finish'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/reporter.rb:81:in `report'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:112:in `run_specs'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:87:in `run'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:71:in `run'
+  /usr/local/bundle/gems/rspec-core-3.7.1/lib/rspec/core/runner.rb:45:in `invoke'
+  /usr/local/bundle/gems/rspec-core-3.7.1/exe/rspec:4:in `<top (required)>'
+  /usr/local/bundle/bin/rspec:29:in `load'
+  /usr/local/bundle/bin/rspec:29:in `<top (required)>'
+```
```

Comment:
That's quite long. I'd see how it is going to look on the page itself to make sure it's not grabbing too much attention away from the post.